src/main/NWN/API/Object/NwCreature.cs

```diff
@@ -356,16 +356,25 @@ public CreatureSize Size
     public float ChallengeRating
     {
       get => NWScript.GetChallengeRating(this);
-    }
-
+    }
+
     /// <summary>
     /// Gets the movement rate of this creature.
     /// </summary>
```

Comment:
Docs need to be update to "Gets the movement rate of this creature."

Reply:
Docs updated